### ISSUE TITLE
Not able to deploy PyTorch Models using Triton Prepackaged Model Server on Openshift

### ISSUE THREAD

**Issue by altruistcoder** [SeldonIO/seldon-core#3019]
I am trying to run the **Triton Prepackaged Model Server** by following the information provided [**here**](https://docs.seldon.io/projects/seldon-core/en/latest/servers/triton.html#example), but its pod is not coming up and I am seeing this error in the corresponding Deployment description:

```
pods "triton-simple-0-simple-86f7c96ffc-" is forbidden: unable to validate against any security context constraint: [spec.containers[0].securityContext.runAsUser: Invalid value: 1000: must be in the ranges: [1000750000, 1000759999]]
```

**Seldon Operator Version: 1.6.0
Cloud Provider: AWS
RedHat Openshift Version: 4.6.8**

I have installed the Seldon Operator from the OperatorHub of my cluster.

The **yaml file** I used as the SeldonDeployment definition file is:

```
apiVersion: machinelearning.seldon.io/v1alpha2
kind: SeldonDeployment
metadata:
  name: triton
spec:
  protocol: kfserving
  predictors:
  - graph:
      implementation: TRITON_SERVER
      modelUri: gs://seldon-models/trtis/simple-model
      name: simple
    name: simple
    replicas: 1
```

Can you please help me in resolving this error?

**Comment by cliveseldon** [comment]
Core issue is https://github.com/triton-inference-server/server/issues/2600

**Comment by cliveseldon** [comment]
One can try to update the namespace you are using to allow any UID:

```
oc adm policy add-scc-to-group anyuid system:serviceaccounts:<namespace>
```

**Comment by cliveseldon** [comment]
This should be fixed via triton PR associated to their [issue](https://github.com/triton-inference-server/server/issues/2600) Please reopen if still an  issue

**Comment by adriangonz** [comment]
Reopening issue to check whether latest Triton image can run with a random UID (and update the operator if it can).

**Comment by mamurak** [comment]
I'm facing the same issue running Seldon Operator 1.13.1 on OpenShift 4.9.

I'm using this SeldonDeployment:
```
apiVersion: machinelearning.seldon.io/v1
kind: SeldonDeployment
metadata:
  name: yolov7
spec:
  name: yolov7
  predictors:
    - graph:
        envSecretRefName: seldon-rclone-secret
        implementation: TRITON_SERVER
        modelUri: 's3://my-bucket/yolov7.pt'
        name: yolov7
        storageInitializerImage: 'seldonio/rclone-storage-initializer:1.12.0'
        type: MODEL
      name: default
      replicas: 1
  protocol: v2
```

The Deployment fails with this error message:
```
pods "yolov7-default-0-yolov7-555667f89f-" is forbidden: unable to
        validate against any security context constraint: [provider "anyuid":
        Forbidden: not usable by user or serviceaccount,
        spec.containers[0].securityContext.runAsUser: Invalid value: 1000: must
        be in the ranges: [1000660000, 1000669999], provider "nonroot":
        Forbidden: not usable by user or serviceaccount, provider
        "hostmount-anyuid": Forbidden: not usable by user or serviceaccount,
        provider "machine-api-termination-handler": Forbidden: not usable by
        user or serviceaccount, provider "hostnetwork": Forbidden: not usable by
        user or serviceaccount, provider "hostaccess": Forbidden: not usable by
        user or serviceaccount, provider "node-exporter": Forbidden: not usable
        by user or serviceaccount, provider "privileged": Forbidden: not usable
        by user or serviceaccount
```

**Comment by adriangonz** [comment]
Hey @mamurak ,

I think the fix was released in Seldon Core `1.14.0`. Would you be able to upgrade to test it out?

**Comment by mamurak** [comment]
Hi @adriangonz,

I'd love to test it on `1.14.0`, but it seems that version is not available for OpenShift yet (#4175). At least it's not available on OperatorHub yet, and the current Helm chart fails to deploy due to SCCs.

**Comment by adriangonz** [comment]
Hey @mamurak ,

The team is currently working on getting the `1.14.0` release up on OpenShift. In the meantime, isn't it possible to just change the operator's image tag to use the `1.14.0` image?

**Comment by mamurak** [comment]
Hi @adriangonz,

As long as `1.14.0` is not available on Operator Hub, OpenShift won't be able to find that operator version:
https://operatorhub.io/operator/seldon-operator
I've tried manipulating the version tag, but that doesn't work as the operator lifecycle is automated and tied to the operator catalog, e.g. the Operator Hub entries.

**Comment by mamurak** [comment]
And the current `1.14.0` operator images that are available through Helm don't work on OpenShift. I've tried that.
I believe much of the team's effort is in making sure the containers comply with OpenShift's security policies.

**Comment by RafalSkolasinski** [comment]
We're in the process of releasing 1.14.0 for OpenShift now https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/1474
Once this is merged you should be able to install 1.14.0 via OperatorHub.
